examples/python/README.md

```diff
@@ -0,0 +1,127 @@
+# Python Examples
+
+This directory contains examples demonstrating how to use the
+Diatheke SDK.
+
+## SDK Installation
+Before the demos can be run, the Python SDK must first be installed.
+The easiest way to do this is to use the pip command to download the 
+SDK module from GitHub:
+```bash
+pip install "git+https://github.com/cobaltspeech/sdk-diatheke#egg=cobalt-diatheke&subdirectory=grpc/py-diatheke"
+
+# Install additional dependencies to run the demos
+pip install toml
+```
+
+Alternatively, it is possible to manually clone the repo from GitHub
+and set the `PYTHONPATH` to point to the correct subdirectory:
+```bash
+# Get the SDK
+git clone https://github.com/cobaltspeech/sdk-diatheke
+
+# Install dependencies
+pip install grpcio-tools
```

Comment:
possible to make this use setup.py? that way the readme won't diverge if deps otherwise change in code.

Reply:
See above on line 11, where it does just that. This is an alternative method if someone doesn't want to install using pip. Would you prefer this alternative method to be removed entirely? Or perhaps you mean something like
```bash
pip install -e /absolute/path/to/sdk-diatheke/grpc/py-diatheke
```

Reply:
I meant, to install deps, not the sdk itself. 

If there a way for pip to read requirements from setup.py?